src/main/java/com/wojtek/parkingmeter/helpers/SumJSON.java

```diff
@@ -1,4 +1,18 @@
 package com.wojtek.parkingmeter.helpers;
 
 public class SumJSON {
+
+    Double sum;
```

Comment:
dlaczego pakietowy dostęp?

Reply:
Faktycznie, zgapiłem. Skoro mam settery i gettery to dałbym private.